docs/en/reference/how_to_setup_metadata_engine.md

```diff
@@ -301,18 +301,18 @@ juicefs mount -d badger://$HOME/badger-data /mnt/jfs
 ```
 
 :::note
-Since BadgerDB is a standalone database, it can only be used locally and does not support multi-host shared mounts. In addition, only one process access is allowed in BadgerDB, and `gc`, `fsck` operations cannot be performed when the file system is mounted.
+Since BadgerDB is a standalone database, it can only be used locally and does not support multi-host shared mounts. In addition, only one process access is allowed in BadgerDB, and `gc` and `fsck` operations cannot be performed when the file system is mounted.
```

Comment:
```suggestion
Since BadgerDB is a standalone database, it can only be used locally and does not support multi-host shared mounts. In addition, only one process is allowed to access BadgerDB all the time, and `gc` and `fsck` operations cannot be performed when the file system is mounted.
```